docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -396,8 +401,9 @@ Install Steward on the cluster (see https://wiki.vshn.net/pages/viewpage.action?
 +
 [source,console]
 ----
-export LIEUTENANT_URL="api.syn.vshn.net"
-export LIEUTENANT_NS="lieutenant-prod"
+export LIEUTENANT_URL="api.syn.vshn.net" # same as COMMODORE_API_URL
+export LIEUTENANT_NS="lieutenant-prod" # or lieutenant-[dev,int] accordingly
+export LIEUTENANT_TOKEN=<lieutenant-api-token> # yes, it's the same as COMMODORE_API_TOKEN
 export LIEUTENANT_AUTH="Authorization:Bearer ${LIEUTENANT_TOKEN}"
```

Comment:
Why not just use `COMMODORE_API_TOKEN` here? The variable `LIEUTENANT_TOKEN` not used anywhere else and do not have to be set.

Reply:
yes, the same with LIEUTENANT_URL actually